src/plugins/telemetry/public/services/telemetry_service.test.ts

```diff
@@ -56,21 +58,25 @@ jest.mock('moment', () => {
     };
   });
 });
-
-describe.skip('TelemetryService', () => {
-  describe.skip('fetchTelemetry', () => {
-    it('calls expected URL with 20 minutes - now', async () => {
-      const telemetryService = mockTelemetryService();
-      await telemetryService.fetchTelemetry();
-      expect(telemetryService['http'].post).toBeCalledWith('/api/telemetry/v2/clusters/_stats', {
-        body: JSON.stringify({ unencrypted: false, timeRange: {} }),
-      });
-      expect(mockClone).toBeCalled();
-      expect(mockSubtract).toBeCalledWith(20, 'minutes');
+*/
+
+describe('TelemetryService', () => {
+  // TODO: enable this unit test when fetchTelemtry function is restored
+  /*
+  describe('fetchTelemetry', () => {
```

Comment:
I think this is a case where we should keep the test skipped instead of commenting it out, and then have an open Issue to address it.